statick_tool/rsc/config.yaml

```diff
@@ -136,12 +136,14 @@ levels:
         flags: ""
       pyflakes:
         flags: ""
+      # Disabling duplicate code as it is giving unreliable false positives when running pylint against multiple files
```

Comment:
Nit: It 
```suggestion
      # Disabling duplicate code as it is giving unreliable results when running pylint against multiple files
```

Reply:
Suggestion is because all the `duplicate-code` results I've seen it give were seemingly valid findings of duplicate code.  We'd consider them "false positives" because using the same 4 `import` lines in multiple files isn't something we'd consider bad.  But it is indeed finding duplicated lines between files - just not consistently.